catalog/spatial/registry/registry-identification-plugin/src/test/java/org/codice/ddf/registry/identification/IdentificationPluginTest.java

```diff
@@ -220,31 +225,102 @@ public void testDuplicateCheckingAfterDelete() throws Exception {
         idp.process(new CreateRequestImpl(sampleData));
     }
 
-    @Test
-    public void testUnusedMethods() throws Exception {
-        UpdateRequest updateRequest = new UpdateRequestImpl("abc123", sampleData);
-        assertThat(updateRequest, equalTo(idp.process(updateRequest)));
-        DeleteRequest deleteRequest = new DeleteRequestImpl("abc123");
-        assertThat(deleteRequest, equalTo(idp.process(deleteRequest)));
-        UpdateResponse updateResponse = new UpdateResponseImpl(updateRequest, null, null);
-        assertThat(updateResponse, equalTo(idp.process(updateResponse)));
-    }
-
     private String convert(String path) throws Exception {
         InputStream inputStream = getClass().getResourceAsStream(path);
         BufferedReader buffer = new BufferedReader(new InputStreamReader(inputStream));
         return buffer.lines()
                 .collect(Collectors.joining("\n"));
     }
 
+    @Test(expected = UnsupportedOperationException.class)
+    public void testUpdateWithNullOperationTransaction() throws Exception {
+        String xml = convert("/registry-both-extid.xml");
+        sampleData.setAttribute(Metacard.METADATA, xml);
+
+        OperationTransaction operationTransaction = new OperationTransactionImpl(null,
+                Collections.singletonList(sampleData));
+        Map<String, Serializable> properties = new HashMap<>();
+        List<Map.Entry<Serializable, Metacard>> updatedEntries = new ArrayList<>();
+
```

Comment:
Shouldn't you code to the interfaces and not the Impls?  It looks like you are doing that a lot in this test.